src/devices/bluetooth.c

```diff
@@ -206,16 +232,31 @@ int bt_init_connection(DeviceConfig *config)
          * factory level (9600).  This ensures things go slow enough for the
          * HC-06 processor + code to handle it.
          */
+        int baud = bt_find_working_baud(config, btConfig);
+        switch (baud) {
+        case 0:
+                pr_info("BT: Failed to communicate with device.  Assuming it "
```

Comment:
so if the BT module is already paired with android you'll arrive on this line.  So the text should say "Assuming already configured, moving on" or some such. 

Because it could either be configured / connected or simply disconnected.  Sigh, HC-06...